src/components/TimeSelectInput/readme.md

```diff
@@ -0,0 +1,78 @@
+##### TimeSelectInput base:
+
+```js
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { TimeSelectInput } from 'react-rainbow-components';
+
+const StyledContainer = styled.div`
+    display: flex;
+    justify-content: center;
+`;
+const TimeSelectInputExample = () => {
+    const [time, setTime] = useState(null); 
+    return ( 
+        <TimeSelectInput 
+            id="time-select-input-2" 
+            value="" 
```

Comment:
since this component is an inputable it must be controlled, then the source of truth is the value prop, then if you don't pass the time state here as the value it should not work